### PR TITLE
Fea: s&box options FOV

### PR DESCRIPTION
**Pull request by mzegar**
Use the FOV in the settings menu of s&box everywhere.

### CODE REVIEW

File: code/player/camera/FirstPersonSpectatorCamera.cs [hunk 1]
```diff
@@ -9,8 +9,6 @@ public partial class FirstPersonSpectatorCamera : Sandbox.Camera, IObservationCa
         public override void Activated()
         {
             base.Activated();
-
-            FieldOfView = 80;
```
Comment (Alf21) [inline code/player/camera/FirstPersonSpectatorCamera.cs:13]: Then, we don't need the Activated method at all.

File: code/player/camera/RagdollSpectateCamera.cs [hunk 2]
```diff
@@ -2,8 +2,50 @@
...
+
+            FocusPoint = CurrentView.Position - GetViewOffset();
+        }
+
+        public override void Update()
+        {
+            if (Local.Client == null)
+            {
+                return;
+            }
```
Comment (Alf21) [inline code/player/camera/RagdollSpectateCamera.cs:21]: Is this needed as it's nowhere used

File: code/player/camera/RagdollSpectateCamera.cs [hunk 3]
```diff
@@ -2,8 +2,50 @@
...
+        }
+
+        public virtual Vector3 GetViewOffset()
+        {
+            if (Local.Client == null)
+            {
+                return Vector3.Zero;
+            }
+
+            return Input.Rotation.Forward * (-130 * 1) + Vector3.Up * (20 * 1);
```
Comment (Alf21) [inline code/player/camera/RagdollSpectateCamera.cs:48]:
```suggestion
            return Input.Rotation.Forward * -130 + Vector3.Up * 20;
```

Or does these `* 1` any sense?

**Review by Alf21 — CHANGES_REQUESTED** [review]

File: code/player/camera/RagdollSpectateCamera.cs (hunk 3 above)
Comment (mzegar) [reply code/player/camera/RagdollSpectateCamera.cs]: Just copied and pasted code from the original ragdoll controller.

**Review by Alf21 — APPROVED** [review]
👍

**Pull request merged**